lib/evaluation/abstractEvaluationHandler.ts

```diff
@@ -21,5 +24,20 @@ export abstract class AbstractEvaluationHandler implements EvaluationHandler {
     return this.nextHandler != undefined;
   }
 
-  public abstract handle(d: PresentationDefinition, p: unknown, results: HandlerCheckResult[]): void;
+  get client(): EvaluationClient {
+    return this._client;
+  }
+
+  set client(client: EvaluationClient) {
+    this._client = client;
+  }
+
+  public abstract handle(d: PresentationDefinition, p: unknown): void;
+
+  get verifiablePresentation(): any {
+    return this.client.verifiablePresentation;
+  }
+  get results(): HandlerCheckResult[] {
```

Comment:
this one can be declared like a simple getter
`getResults()`

see related above comment in the super / interface as well.